.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
     hooks:
     -   id: flake8
         language: python_venv
-        additional_dependencies: [flake8-comprehensions]
+        additional_dependencies: [flake8-comprehensions>=3.1.0]
```

Comment:
you should also change in environment.yaml

Reply:
Thanks @jreback done.